sponsor-dapp-v2/src/components/common/Dropdown.js

```diff
@@ -63,14 +72,14 @@ class Dropdown extends Component {
           {list.map((item, index) => {
             return (
               <li
-                key={index}
+                key={item.key}
                 value={index}
```

Comment:
For my own curiosity, what is this `value` field?

Reply:
No idea - I only know the key field is supposed to be the "unique identifier that the list uses to refer to elements. I didn't touch the value because I wasn't able to find any documentation about what it does :/